haystack/testing/document_store.py

```diff
@@ -538,6 +538,20 @@ def test_custom_embedding_field(self, ds):
         # Some document stores normalize the embedding on save, let's just compare the length
         assert doc_to_write["custom_embedding_field"].shape == documents[0].embedding.shape
 
+    # TODO This currently fails for Weaviate and Pinecone
+    # @pytest.mark.integration
+    # @pytest.mark.parametrize("batch_size", [None, 20])
```

Comment:
Hey @julian-risch I opted to leave this here so we can visit in a future PR to reactivate. What do you think?

Reply:
Fair enough for this PR. 👍 I will review the PR later today.